### PR TITLE
SPLAT-872 - doc(release-stable): add manual steps to release to stable

### PR DESCRIPTION
**Pull request by mtulio**
Add manual steps to promote release to `stable`.

https://issues.redhat.com/browse/SPLAT-872

### CODE REVIEW

File: openshift-tests-provider-cert/README.md [hunk 1]
```diff
@@ -58,6 +58,48 @@ git push origin v0.0.0-demo1
...
+
+The steps to promote a release to stable are manual, please follow those steps:
+
+1. Create a cluster on 4.11+ (usually in AWS)
+1. Add a dedicated node (optional): `./hack/oc-create-machineset-aws-dedicated.sh`
+1. Clone the [CLI/openshift-provider-cert](https://github.com/redhat-openshift-ecosystem/provider-certification-tool) repository, replacing the tag `stable` for [each plugin manifest](https://github.com/redhat-openshift-ecosystem/provider-certification-tool/tree/main/manifests) to the new tag to be evaluated (Example: `v0.1.0`)
+```bash
+make update
+make build
+```
```
Comment (bostrt) [inline openshift-tests-provider-cert/README.md:73]: Is this run in the plugin or tools repo?

Comment (mtulio) [reply openshift-tests-provider-cert/README.md:73]: Tools repo.

Comment (mtulio) [reply openshift-tests-provider-cert/README.md:73]: I am not sure, that's the way I am following. But avoiding building could be better. So will the steps be like this?:
```
./openshift-provider-cert-linux-amd64 assets
./openshift-provider-cert-linux-amd64 run -w --dedicated \
  --plugin=./openshift-kube-conformance_env-shared.yaml \
  --plugin=./openshift-conformance-validated_env-shared.yaml \
```

**Comment by mtulio** [comment]
@bostrt ptal?

File: openshift-tests-provider-cert/README.md [hunk 2]
```diff
@@ -58,6 +58,50 @@ git push origin v0.0.0-demo1
...
+  --plugin=./openshift-kube-conformance_env-shared.yaml \
+  --plugin=./openshift-conformance-validated_env-shared.yaml \
+```
+1. Make sure the dedicated node is running, ROLE=tests (`node-role.kubernetes.io/tests=''`): `oc get nodes`
+1. Run the certification environment: `./openshift-provider-cert-linux-amd64 run -w --dedicated`
+1. Check if the plugin image was created correctly with your new release: `oc describe pods -n openshift-provider-certification | grep Image:`
+1. Wait for the tests to be finished.
+1. Once the execution has been finished, the post processor should display the results, somehting like `Total tests processed: 1837 (1777 pass / 60 failed)` (valid counters)
+1. Collect the archive `./openshift-provider-cert-linux-amd64 retrieve`
+1. Inspect the results ``./openshift-provider-cert-linux-amd64 results <artchive>.tar.gz`
```
Comment (bostrt) [inline openshift-tests-provider-cert/README.md:82]: Remove extra grave accent ` here

**Comment by bostrt** [comment]
I have one nit but otherwise /lgtm

File: openshift-tests-provider-cert/README.md (hunk 2 above)
Comment (mtulio) [inline openshift-tests-provider-cert/README.md:82]:
```suggestion
1. Inspect the results `./openshift-provider-cert-linux-amd64 results <artchive>.tar.gz`
```

Comment (mtulio) [reply openshift-tests-provider-cert/README.md]: Done, thanks

**Comment by mtulio** [comment]
> I have one nit but otherwise /lgtm

Thanks, Robert, reviewed. Ptal and set the `lgtm` label?

**Review by bostrt — APPROVED** [review]

**Pull request merged**